server-mock/server/get-whitelist.go

```diff
@@ -1,15 +1,45 @@
 package server
 
 import (
+	"fmt"
 	"net/http"
 
+	"github.com/ethereum/go-ethereum/crypto"
 	apiv2 "github.com/nodeset-org/nodeset-client-go/api-v2"
+	"github.com/nodeset-org/nodeset-client-go/server-mock/db"
+	nsutil "github.com/nodeset-org/nodeset-client-go/utils"
+	"github.com/rocket-pool/node-manager-core/utils"
 )
 
 func (s *NodeSetMockServer) getWhitelist(w http.ResponseWriter, r *http.Request) {
 	data := apiv2.WhitelistData{}
+
+	// Get the requesting node
+	session := s.processAuthHeader(w, r)
+	if session == nil {
+		return
+	}
+	node := s.getNodeForSession(w, session)
+	if node == nil {
+		return
+	}
+
+	db := db.NewDatabase(s.logger)
+	privateKey, err := crypto.ToECDSA(db.ConstellationAdminPrivateKey)
+	if err != nil {
+		fmt.Printf("error converting private key: %w", err)
+		return
+	}
+
+	adminAddress := crypto.PubkeyToAddress(privateKey.PublicKey).Hex()
+	message := []byte(node.Address.Hex() + adminAddress)
```

Comment:
I don't think you want to use `node.Address.Hex()` here, I think you want `node.Address[:]` to get the raw address bytes instead of the ASCII bytes.

Reply:
updated